docker/keycloak-docker-compose.yml

```diff
@@ -6,12 +6,14 @@ version: "2"
 
 services:
   keycloak:
-    image: "quay.io/keycloak/keycloak:15.0.0"
+    image: "quay.io/keycloak/keycloak:19.0.0"
```

Comment:
Is there a reason we're only going to 19?

Reply:
That's what we're running in staging/production at the moment, and I didn't think it would make sense to push into uncharted version territory in the example dev environment setup

Reply:
Never mind, just saw the Jira ticket you created regarding this!